### PR TITLE
Add GUI test for help window #70

### LINKED ISSUE #70: Add GUI test for help window
As per title

### PR DESCRIPTION
**Pull request by yl-coder**
Fixes #70

### CODE REVIEW

File: src/test/java/guitests/guihandles/MainMenuHandle.java [hunk 1]
```diff
@@ -18,4 +20,19 @@ public GuiHandle clickOn(String... menuText) {
         Arrays.stream(menuText).forEach((menuItem) -> guiRobot.clickOn(menuItem));
         return this;
     }
+
+    public HelpWindowHandle openHelpWindowUsingMenu() {
+        clickOn(new String[] {"Help", "F1"});
```
Comment (damithc) [inline src/test/java/guitests/guihandles/MainMenuHandle.java]: There's no `clickOn(String... menuItems)`? That way, you can write `clickOn("Help", "F1")`

File: src/test/java/guitests/guihandles/MainMenuHandle.java [hunk 2]
```diff
@@ -18,4 +20,19 @@ public GuiHandle clickOn(String... menuText) {
...
+        return new HelpWindowHandle(guiRobot, primaryStage);
+    }
+
+    public HelpWindowHandle openHelpWindowUsingAccelerator() {
+        use_F1_accelerator();
+        return new HelpWindowHandle(guiRobot, primaryStage);
+    }
+
+    private void use_F1_accelerator() {
```
Comment (damithc) [inline src/test/java/guitests/guihandles/MainMenuHandle.java]: Remove underscores?

File: src/test/java/guitests/guihandles/MainMenuHandle.java (hunk 1 above)
Comment (yl-coder) [inline src/test/java/guitests/guihandles/MainMenuHandle.java]: done

File: src/test/java/guitests/guihandles/MainMenuHandle.java (hunk 2 above)
Comment (yl-coder) [inline src/test/java/guitests/guihandles/MainMenuHandle.java]: done

**Pull request merged**